Dockerfile

```diff
@@ -3,8 +3,8 @@ RUN dnf install -y cargo openssl-devel xz-devel
 WORKDIR /build
 COPY Cargo.* ./
 COPY src src/
-# Disable LTO to avoid OOM in Quay
-RUN sed -i 's/^lto = true$/lto = false/' Cargo.toml
+# Debug symbols are nice but they're not 112 MB of nice
```

Comment:
completely optional: Make the comment more approximate since the actual space saving could change over time. 

```suggestion
# Debug symbols are nice but they're not 100+ MB of nice
```